cypress/integration/tests/controls/stakeholdergroups/create.test.ts

```diff
@@ -19,17 +19,17 @@ import {
 import { Stakeholdergroups } from "../../../models/stakeholdergroups";
 
 import * as commonView from "../../../../integration/views/commoncontrols.view";
-import * as faker from "faker";
+import * as data from "../../../../utils/data_utils";
 
 describe("Basic checks while creating stakeholder groups", () => {
```

Comment:
Update test case names as per these changes - https://github.com/konveyor/tackle-ui-tests/pull/12/files#diff-a94b88b529d0d6aed9a45d31accc2018e7ca725418aceed5c10472a819028f36R30
@sshveta has suggested it. and it looks very recognizable.

Reply:
Sounds good, will update for all